README.md

```diff
@@ -83,7 +83,7 @@ On macOS and Linux, you can also use [Homebrew](https://brew.sh/) to install `zq
 ```
 brew install brimdata/tap/zq
 ```
-To install `zed`, use
+To install `zed` for working with lakes, use
```

Comment:
Maybe these extra words are overkill. But I suggested adding them because the above text in this README does a nice job of setting expectations that `zq` is a standalone command-line tool because it's compared to `jq`. However, it's not as clearly spelled out that `zed` the command line tool is the one for lakes. But since the word "Zed" appears so many times already in this doc, I figure a little extra disambiguation here can't hurt. It's admittedly all made even more clear in the lines that follow.

Reply:
I also think "use" should be "run" here